Networking/Networking/Model/Order.swift

```diff
@@ -156,6 +159,7 @@ public struct Order: Decodable, GeneratedCopiable, GeneratedFakeable {
         let status = try container.decode(OrderStatusEnum.self, forKey: .status)
 
         let currency = try container.decode(String.self, forKey: .currency)
+        let currencySymbol = try container.decode(String.self, forKey: .currencySymbol)
```

Comment:
nit: might be worth adding a line of test in `OrderMapperTests.test_Order_fields_are_properly_parsed`?